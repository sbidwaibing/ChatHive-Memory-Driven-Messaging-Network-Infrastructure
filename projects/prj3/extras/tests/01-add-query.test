+ @john room-01 #sqlite #db
sqlite is cool
.
? room-01
.
