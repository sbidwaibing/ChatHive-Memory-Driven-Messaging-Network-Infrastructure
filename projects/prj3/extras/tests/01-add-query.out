ok
ok
YYYY-MM-DDThh:mm:ss.ttt
@john room-01 #db #sqlite
sqlite is cool
